### ISSUE TITLE
Pipewire audio in GDM not working

### ISSUE THREAD

**Issue by destructatron** [NixOS/nixpkgs#294842]
### Describe the bug
When using Pipewire as the audio server, audio in GDM does not work. Audio is needed so that the Orca screen reader can start within GDM and for the login screen to be accessible.

### Steps To Reproduce
1. Power on and wait for GDM to appear.
2. Press super alt S. Orca won't speak.
3. Using volume keys and muting and unmuting will fail to get audio going.

### Expected behavior
Audio should be working within GDM, and I should be able to successfully use Orca to log into a desktop session.

### Additional context
Lightdm's configuration allows for pipewire and for audio to start, and Orca is able to be used. On other Linux distributions, audio in GDM has always worked.

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[harley@harley-laptop:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.7.7-xanmod1, NixOS, 24.05 (Uakari), 24.05pre594677.3030f185  - host os: `Linux 6.7.7-xanmod1, NixOS, 24.05 (Uakari), 24.05pre594677.3030f185
ba6a`
  - multi-user?: `yes`
  - sandbox: `yes`
  - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"home-manager, nixos"`
  - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by destructatron** [comment]
@prusnak
